hooks/post_gen_project.py

```diff
@@ -4,33 +4,59 @@
 
 print("Running post generation...")
 
-ci = "{{ cookiecutter.ci }}"
+files_to_be_removed = []
```

Comment:
why isn't it written in uppercase? It is a global variable as well

Reply:
But it's not a typical constant because it is getting mutated later down the track. Uppercase is for constants (hyperparameters) not for any global variables, at least for me.